Core/Core/Planner/APIPlannable.swift

```diff
@@ -146,3 +146,19 @@ public struct GetPlannablesRequest: APIRequestable {
         ]
     }
 }
+
+// https://canvas.instructure.com/doc/api/planner.html#method.planner_notes.create
+public struct CreatePlannerNote: APIRequestable {
```

Comment:
`PostPlannerNoteRequest` the convention for `APIRequestable` is `{HTTP verb}{thing}Request`. A use case would be named `CreatePlannerNote`, not a request.